spec/support/shared_examples_policy_machine.rb

```diff
@@ -370,6 +370,17 @@
       policy_machine.is_privilege?(@u1, @w, @o1).should be_false
     end
 
+    context 'tolerate_cycles is set to true' do
```

Comment:
Can the postgres adapter be excluded with a conditional but have a test for the mysql and in memory adapters